Cargo.toml

```diff
@@ -49,12 +49,12 @@ atomic-waker = "1.1.2"
 base64 = "0.22.1"
 bitfield = "0.17.0"
 bytecodec = "0.4.15"
-bytes = "1.7.2"
-cargo_metadata = "0.18.1"
+bytes = "1.8.0"
+cargo_metadata = "0.19.0"
 cbindgen = "0.27.0"
 chrono = "0.4.38"
 chrono-tz = "0.10.0"
-clap = "4.5.20"
+clap = "4.5.21"
```

Comment:
⬇️ - I had hoped to upgrade `embedded-svc` and `esp-idf-svc`, but did not succeed due to a puzzling error. I'll take another go at it, since I believe all that was blocking it was the rust version.